---
title: "'Finally' no puede aparecer fuera de una instrucción 'Try'"
ms.date: 07/20/2015
f1_keywords:
- vbc30382
- bc30382
helpviewer_keywords:
- BC30382
ms.assetid: 0314d8d2-18bc-4bbd-858c-0a18408b52fd
ms.openlocfilehash: 94461fece317e01d63dfb13f082b07165454869a
ms.sourcegitcommit: bf5c5850654187705bc94cc40ebfb62fe346ab02
ms.translationtype: MT
ms.contentlocale: es-ES
ms.lasthandoff: 09/23/2020
ms.locfileid: "91074516"
---
# <a name="finally-cannot-appear-outside-a-try-statement"></a>'Finally' no puede aparecer fuera de una instrucción 'Try'

`Finally` se usa para completar un bloque `Try...Catch...Finally` ; por lo tanto, solo puede aparecer una vez al final del bloque. O bien hay un elemento `Finally`innecesario, o la instrucción `Finally` aparece fuera de los límites de su bloque `Try` correspondiente.  
  
 **Id. de error:** BC30382  
  
## <a name="to-correct-this-error"></a>Para corregir este error  
  
1. Busque y quite la instrucción `Finally` innecesaria.  
  
2. Mueva la instrucción `Finally` a la ubicación adecuada del código.  
  
## <a name="see-also"></a>Vea también

- [Try... Detectar... Finally (instrucción)](../language-reference/statements/try-catch-finally-statement.md)
